skopt/gp_opt.py

```diff
@@ -55,37 +55,30 @@ def acquisition(X, model, x_opt=None, method="UCB", xi=0.01, kappa=1.96,
     if X.ndim == 1:
         X = np.expand_dims(X, axis=0)
 
-    # Rescale to between 0 and 1.
-    if bounds is not None:
-        lower_bounds, upper_bounds = extract_bounds(bounds)
-        X = (X - lower_bounds) / (upper_bounds - lower_bounds)
-
     # Compute posterior
     mu, std = model.predict(X, return_std=True)
+    mu = mu.ravel()
+    std = std.ravel()
 
     # Evaluate acquisition function
     if method == "UCB":
         values = mu - kappa * std
 
     elif method == "EI":
-        improvement = x_opt - mu
-        exploit = improvement * stats.norm.cdf(improvement / std)
+        improvement = mu - x_opt - xi
+        exploit = std * improvement * stats.norm.cdf(improvement / std)
```

Comment:
From Shahriari et al "Taking the Human out of the loop" eqn 44 I think it should be just `improvement * stats.norm.cdf(improvement / std)`.

I agree on L68 where the order of `mu` and `x_opt` has been swapped.

Do you have a reference with some details on the ideas behind subtracting `xi`?